<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MDXfind-GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>MDXfind</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Hash file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="hashFile"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectHashFile">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Hash types:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="preHashTypes">
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DYN1024</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HAV128_4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HAV128_5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HAV</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>LM</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5-4xMD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5BASE64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5HEXSALT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5MD5SALT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5PASSMD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5rev</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5SALT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5SALTPASS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5SHA1SHA1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5SHA1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5SQL5-32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5USERIDMD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5USERIDMD5MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NTLM</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>RMD128</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA1BASE64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA1MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA1RAW</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA256</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA512</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SQL5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TIGER</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>WRL</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>salted MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>salted SHA1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>all quick MD5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>all quick SHA1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>all quick</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>all WRL</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>all</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="hashTypes"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="setSaltFile">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Provide own salt file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="saltFile"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectSaltFile">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="setUserIDFile">
              <property name="text">
               <string>Provide own userid file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userIDFile"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectUserIDFile">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="setRotation">
              <property name="text">
               <string>Use rotation:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rotation"/>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setAppendNum">
              <property name="text">
               <string>Append numbers (dec/hex):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="appendNum"/>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setIterations">
              <property name="text">
               <string>Iterations:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="iterations"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCheckBox" name="setReplaceXML">
              <property name="text">
               <string>Replace unusual characters with XML</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setAddCLRF">
              <property name="text">
               <string>Add CR/LF/... to begin and end</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setUseUnicode">
              <property name="text">
               <string>Use Unicode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="setNumThreads">
              <property name="text">
               <string>Set num threads:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="numThreads"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QCheckBox" name="setUseRules">
              <property name="text">
               <string>Use rules:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="rules"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectRules">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="setPermutationFile">
              <property name="text">
               <string>Permutation file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="permutationFile"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectPermutationFile">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QCheckBox" name="setSaveFounds">
              <property name="text">
               <string>Save founds to file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="saveFounds"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectSaveFounds">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>MDXfind location:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mdxfindLocation"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectMdxfindLocation">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QCheckBox" name="setClean">
              <property name="text">
               <string>Create clean left/found:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="setCleanPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectClean">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="addToQueue">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add to queue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="startProcess">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="info">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Wordlists</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Not used:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Used:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QListWidget" name="wordlistsNo">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Here all wordlists are listed which will &lt;span style=&quot; font-weight:600;&quot;&gt;not&lt;/span&gt; be processed by MDXfind&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="wordlistsYes">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Here all wordlists are listed which will be processed by MDXfind&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="wordlistsLeft">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Move selected wordlists from right to left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wordlistsAdd">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add wordlists&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wordlistsRemove">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove selected wordlists (From both lists!)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Remove selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wordlistsRight">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Move selected wordlists from left to right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Queue</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="queue"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QPushButton" name="queueClear">
            <property name="text">
             <string>Clear queue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="queueStart">
            <property name="text">
             <string>Process queue</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;MDXfind-GUI&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Tool created by s3in!c to simplify the use of the great tool MDXfind from Waffle.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;You need the MDXfind application additional to this GUI to be able to work. More information about the usage of MDXfind and MDXfind-GUI can be found here: &lt;/span&gt;http://hashes.org/forum/forumdisplay.php?fid=14&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
